src/puppetlabs/trapperkeeper/services/status/ringutils.clj

```diff
@@ -0,0 +1,65 @@
+;; TODO: Much of this code is copied from other projects. It should
+;; probably be moved into a shared library or some such.
```

Comment:
I copied this straight out of file sync service. Perhaps we should add these to https://github.com/puppetlabs/ring-middleware?